client/src/components/Navbar/Navbar.js

```diff
@@ -6,6 +6,14 @@ import blackWave from "./side-bar.svg";
 
 function Navbar() {
   
+  React.useEffect(() => {
```

Comment:
neat implementation!
You forgot to unregister this!

Reply:
https://github.com/mozilla/FirefoxColor/pull/862/files#diff-13cb0431174110511a60cfb1f77968ddR12-R26
Check this for reference!